src/main/java/com/softserve/academy/spaced/repetition/controller/ImageController.java

```diff
@@ -26,7 +26,7 @@
 public class ImageController {
 
     @Autowired
-    private ImageService imageService;
+    private ImageServiceImpl imageServiceImpl;
```

Comment:
Use interface to inject bean. (ImageServiceImpl -> ImageService) and change variable name (imageServiceImpl -> imageService). Check in all classes